src/protocol.rs

```diff
@@ -287,7 +287,18 @@ impl ParticipantConfig {
     }
 }
 
-/// An identifier corresponding to a [`Participant`].
+/// An identifier for a [`Participant`].
+///
+/// All [`Participant`]s in a session must agree on the
+/// [`ParticipantIdentifier`]s. That is, these are not local identifiers for a
+/// single [`Participant`]; they are global identifiers for the [`Participant`]s
+/// in a session.
+///
+/// [`ParticipantIdentifier`]s can be used across multiple sessions. For
+/// example, if a set of participants run keygen, auxinfo, and then compute
+/// several signatures, they can use the same set of identifiers for each of
+/// those sessions. However, we do not recommend reusing identifiers across
+/// different participant sets or for different entities.
```

Comment:
Are participant identifiers meant to tie into how the calling application realizes message authentication? (An unstated assumption on networking is that the participants are connected via authenticated point-to-point channels and have access to an authenticated broadcast mechanism.)

Reply:
@amaloz [just brought this up, too](https://github.com/boltlabs-inc/tss-ecdsa/pull/278#discussion_r1165920807). They do tie into message authentication, but that's a requirement for the calling application. I was going to do a separate PR that focuses on all the calling application requirements around networking (e.g. defining what type of channels they need to support and what kind of message validation they need to do).

Reply:
Made #294 to address this.